### PR TITLE
Constant propagation for ONNXMin, ONNXMax and ONNXSum

### PR DESCRIPTION
**Pull request by tungld**
This patch adds a template for constant propagation for variadic element-wise operations, and applies the template to ONNXMin, ONNXMax and ONNXSum.

### CODE REVIEW

File: src/Transform/ONNX/ConstProp.cpp [hunk 1]
```diff
@@ -238,6 +238,33 @@ Value ConstPropElementwiseBinary(PatternRewriter &rewriter,
   return createReplacingConstantOp(rewriter, replacingValue, resultElements);
 }
 
+/// Do element-wise binary calculation of a variadic value and create an
+/// ONNXConstantOp for the result.
+template <typename ElementwiseBinaryOp>
+Value ConstPropVariadicElementwiseBinary(
+    PatternRewriter &rewriter, Value replacingValue, ValueRange inputList) {
+  assert(inputList.size() > 0 && "The variadic input is emply");
```
Comment (philass) [inline src/Transform/ONNX/ConstProp.cpp:246]: "emply" -> "empty"

**Review by philass — APPROVED** [review]
LGTM!

It seems that you can also get const prop for ONNXSumOp using the same boiler plate https://github.com/onnx/onnx/blob/main/docs/Changelog.md#Sum-13.

File: src/Transform/ONNX/ConstProp.cpp (hunk 1 above)
Comment (tungld) [reply src/Transform/ONNX/ConstProp.cpp]: Thanks! I fixed it.

**Comment by tungld** [comment]
> It seems that you can also get const prop for ONNXSumOp using the same boiler plate https://github.com/onnx/onnx/blob/main/docs/Changelog.md#Sum-13.

Yes, I added ONNXSumOp.

**Review by sorenlassen — APPROVED** [review]
LGTM

**Pull request merged**